plugins/ltac/taccoerce.ml

```diff
@@ -447,7 +449,7 @@ exception CoercionError of Id.t * (Environ.env * Evd.evar_map) option * Val.t *
 let () = CErrors.register_handler begin function
 | CoercionError (id, env, v, s) ->
   Some (str "Ltac variable " ++ Id.print id ++
-   strbrk " is bound to" ++ spc () ++ pr_value env v ++ spc () ++
+   strbrk " is bound to" ++ spc () ++ pr_value ~and_type:true env v ++ spc () ++
```

Comment:
here

Reply:
But this is not used in pr_value? I don't understand why you add this argument.